frontend/src/host_orchestrator/orchestrator/controller.go

```diff
@@ -76,6 +76,9 @@ func (c *Controller) AddRoutes(router *mux.Router) {
 	// reached. Be prepared to retry if the deadline was reached.
 	router.Handle("/operations/{name}/:wait",
 		httpHandler(&waitOperationHandler{c.OperationManager, c.WaitOperationDuration})).Methods("POST")
+	router.Handle("/cvdbugreports",
+		httpHandler(newCreateCVDBugReportHandler(c.Config, c.OperationManager))).Methods("POST")
+	router.Handle("/cvdbugreports/{uuid}/:download", &downloadCVDBugReportHandler{c.Config}).Methods("POST")
```

Comment:
why POST? Does it delete it from the remote host?

Reply:
good catch, just carried the POST, changed it to GET.